jablib/src/main/java/org/jabref/logic/importer/fetcher/ScienceDirect.java

```diff
@@ -137,7 +137,7 @@ private String getUrlByDoi(String doi) throws UnirestException {
         try {
             String request = API_URL + doi;
             HttpResponse<JsonNode> jsonResponse = Unirest.get(request)
-                                                         .header("X-ELS-APIKey", importerPreferences.getApiKey(getName()).orElse(""))
+                                                         .header("X-ELS-APIKey", importerPreferences.getApiKey("Scopus").orElse(""))
```

Comment:
Please add a comment that the keys are shared. 

Add a link to `docs/code-howtos/fetchers.md`